docs/howto/danceable-lyrics-input-tool/README.md

```diff
@@ -281,99 +282,97 @@ def on_complete(self) -> None:
             "views",
         )
         .filter(
-            pl.col("lyrics").str.contains("(?i)" + "|(?i)".join(self.LYRICS_TERMS))
+            (pl.col("views") > self.MIN_VIEWS)
+            & (pl.col("language") == "en")
+            & (
+                pl.col("lyrics").str.contains(
+                    "(?i)" + "|(?i)".join(self.LYRICS_TERMS)
+                )
+            )
         )
-        .filter(pl.col("language") == "en")
-        .filter(pl.col("views") > self.MIN_VIEWS)
-        .collect()
+        .select("track_name", "artist_name")
     )
 ```
 
-Above, we tell Polars to create a [DataFrame](https://pola-rs.github.io/polars/py-polars/html/reference/dataframe/index.html) that contains select columns from rows that match our criteria. We also normalize some column names— we lowercase them and add aliases in our output. In our `filter` clause, we build a case-insensitive [regular expression](https://en.wikipedia.org/wiki/Regular_expression) from our list of terms. Finally, we filter out non-English terms and restrict results to only entries with more than `self.MIN_VIEWS`. You might want to alter these criteria a bit. For example, you might want to allow other languages.
+Above, we tell Polars to create a [LazyFrame](https://pola-rs.github.io/polars/py-polars/html/reference/lazyframe/index.html) that contains select columns from rows that match our criteria. We also normalize some column names— we lowercase them and add aliases in our output. In our `filter` clause, we build a case-insensitive [regular expression](https://en.wikipedia.org/wiki/Regular_expression) from our list of terms. We then filter out non-English terms and restrict results to only entries with more than `self.MIN_VIEWS`. Finally, we only use `track_name` and `artist_name` in the query. You might want to alter these criteria a bit. For example, you might want to allow other languages.
```

Comment:
TW review:

Above, we tell Polars to create a [LazyFrame](https://pola-rs.github.io/polars/py-polars/html/reference/lazyframe/index.html) that contains select columns from rows that match our criteria. We also normalize some column names—we lowercase them and add aliases in the output. In the `filter` clause, we build a case-insensitive [regular expression](https://en.wikipedia.org/wiki/Regular_expression) from the list of terms. We then filter out non-English terms and restrict results to only entries with more than `self.MIN_VIEWS`. Finally, we only use `track_name` and `artist_name` in the query. You might want to alter these criteria a bit. For example, you might want to allow other languages.